### PR TITLE
Improve death screen and timer

### PR DESCRIPTION
**Pull request by toberge**
better death

### CODE REVIEW

File: Assets/Scripts/Gamestate/MatchController.cs [hunk 1]
```diff
@@ -110,11 +114,17 @@ public void EndActiveRound()
 
         if (!IsWin())
         {
-            changeInputMappings("FPS");
```
Comment (Fueredoriku) [inline Assets/Scripts/Gamestate/MatchController.cs:113]: Nice 👍

**Review by Fueredoriku — CHANGES_REQUESTED** [review]
Read code and tested. Looks nice!
Made a small request comment, but other than that I have no other comments :)

File: Assets/Scripts/Gamestate/MatchController.cs (hunk 1 above)
Comment (toberge) [reply Assets/Scripts/Gamestate/MatchController.cs:113]: Thanks

**Review by Fueredoriku — APPROVED** [review]
Looks good! 😎👍

**Pull request merged**